sdcm/fill_db_data.py

```diff
@@ -3019,11 +3055,15 @@ def cql_create_tables(self, session):
             # As of now it behaves as "run_condition".
             if not item['skip'] and ('skip_condition' not in item or eval(str(item['skip_condition']))):
                 for create_table in item['create_tables']:
+                    if self.version_cdc_support():
+                        create_table = self._enable_cdc(item, create_table)
                     # wait a while before creating index, there is a delay of create table for waiting the schema agreement
                     if 'CREATE INDEX' in create_table.upper():
                         time.sleep(15)
                     self.log.debug(f"create table: {create_table}")
                     session.execute(create_table)
+                    # sleep for 15 seconds to wait creating cdc tables
+                    time.sleep(15)
```

Comment:
isn't there a way to check if the cdc tables were created other than blindly sleep?

Reply:
Added wait_schema_agreement()